### PR TITLE
test(e2e): Add AWS e2e tests

### PR DESCRIPTION
**Pull request by jimmidyson**
These can be run via:

```
make e2e-test E2E_LABEL='provider:AWS && cni:Cilium && addonStrategy:ClusterResourceSet'
```

Blocked by #407.
Blocked by #413.

### CODE REVIEW

File: docs/content/addons/_index.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 +++
 title = "Addons"
+weight = 2
+icon = "fa-solid fa-cubes"
```
Comment (faiq) [inline docs/content/addons/_index.md:4]: please split these out to another PR

Comment (jimmidyson) [reply docs/content/addons/_index.md]: Done in #414.
